sky/engine/core/script/dart_controller.cc

```diff
@@ -106,6 +106,15 @@ void DartController::RunFromSnapshot(
       base::Bind(&DartController::DidLoadSnapshot, weak_factory_.GetWeakPtr()));
 }
 
+void DartController::RunFromSnapshotBuffer(const uint8_t* buffer, size_t size) {
```

Comment:
Is there a reason RunFromSnapshot doesn't eventually call RunFromSnapshotBuffer?